mmv1/products/compute/RegionSecurityPolicy.yaml

```diff
@@ -193,3 +197,495 @@ properties:
             Encoded as a hexadecimal number (starting with "0x").
             The last byte of the field (in network byte order) corresponds to the least significant byte of the mask.
           min_version: 'beta'
+  - name: 'rules'
+    type: Array
+    description: |
+      The set of rules that belong to this policy. There must always be a default rule (rule with priority 2147483647 and match "*"). If no rules are provided when creating a security policy, a default rule with action "allow" will be added.
+    min_version: 'beta'
```

Comment:
Just want to double-check that these fields are beta? I don't see any indication of that in a quick scan of the API docs. `rules` is also present in the GA docs: https://cloud.google.com/compute/docs/reference/rest/v1/regionSecurityPolicies